zuul-core/src/main/java/com/netflix/zuul/netty/server/push/PushRegistrationHandler.java

```diff
@@ -95,10 +95,9 @@ public final void channelInactive(ChannelHandlerContext ctx) throws Exception {
     }
 
     @Override
-    public final void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
         logger.error("Exception caught, closing push channel for " + authEvent, cause);
         ctx.close();
```

Comment:
Do we also need to `closeOnFlush` here?

```
if (ch.isActive()) {
    ch.writeAndFlush(Unpooled.EMPTY_BUFFER).addListener(ChannelFutureListener.CLOSE);
}
```

Reply:
Synced offline — we can keep this change small and keep the `close()` behavior while still removing the `final` in case we (or anyone else) wants to override with the fancier `closeOnFlush` 👍